doc/user/content/cli/_index.md

```diff
@@ -94,3 +98,63 @@ responsibility of the network firewall to limit incoming connections. If you
 wish to configure `materialized` to only listen to, e.g. localhost connections,
 you can set `--listen-addr` to `localhost:6875`. You can also use this to change
 the port that Materialize listens on from the default `6875`.
+
+### TLS encryption
+
+Materialize can use Transport Layer Security (TLS) to encrypt traffic between
+SQL and HTTP clients and the `materialized` server.
+
+#### Configuration
+
+To enable TLS, you will need to supply two files, one containing a TLS
```

Comment:
This reads really well! My inclination for any kind of enumeration is to make things a list (or a table); not a big deal but if you like the idea, give it a try. If not, ignore.